site/content/docs/user/ingress.md

```diff
@@ -47,116 +57,31 @@ Apply the [mandatory ingress-nginx components](https://kubernetes.github.io/ingr
 ```shell script
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml
 ```
-Apply kind specific patches
-
-```yaml
-spec:
-  template:
-    spec:
-      containers:
-      - name: nginx-ingress-controller
-        ports:
-        - containerPort: 80
-        # Proxy the host port 80 for http
-          hostPort: 80
-        - containerPort: 443
-        # Proxy the host port 443 for https
-          hostPort: 443
-      nodeSelector:
-        # schedule it on the control-plane node
-        ingress-ready: 'true'
-      tolerations:
-      # tolerate the the control-plane taints
-      - key: node-role.kubernetes.io/master
-        operator: Equal
-        effect: NoSchedule
-```
+Apply kind specific patches to forward the hostPorts to the 
+ingress controller, set taint tolerations and 
+schedule it to the custom labelled node.
+
+{{< codefromfile file="assets/examples/ingress/nginx/patch.json" lang="json" >}}
 
 ```shell script
-kubectl patch deployments -n ingress-nginx nginx-ingress-controller -p "$(curl https://kind.sigs.k8s.io/manifests/ingress/nginx/patch.yaml)"
+kubectl patch deployments -n ingress-nginx nginx-ingress-controller -p '{{< minify file="examples/ingress/nginx/patch.json" >}}' 
 ```
 
 
-Now you will want to checkout [Using Ingress](#using-ingress)
-
+Now the Ingress is all setup to be used. 
+Refer [Using Ingress](#using-ingress) for a basic example usage.
 
 ## Using Ingress
 
 The following example creates simple http-echo services 
 and an Ingress object to route to these services.
 
-```yaml
-kind: Pod
-apiVersion: v1
-metadata:
-  name: foo-app
-  labels:
-    app: foo
-spec:
-  containers:
-    - name: foo-app
-      image: hashicorp/http-echo
-      args:
-        - "-text=foo"
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: foo-service
-spec:
-  selector:
-    app: foo
-  ports:
-    - port: 5678 # Default port used by the image
```

Comment:
Nit thought: We should put the comments on their own lines to make it less wide.